src/style.css

```diff
@@ -1,3 +1,108 @@
+/* Reset de margens padrão do navegador */
+body, h1, h2, h3, h4, h5, h6, p, ul, ol, li, figure, figcaption, blockquote, dl, dd {
+  margin: 0;
+}
+
+/* Importação da fonte Roboto */
 body {
+  background-color: rgb(230, 227, 227);
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  display: flex;
+  justify-content: center; /* Centraliza horizontalmente */
+  align-items: center; /* Centraliza verticalmente */
+}
+
+/* Estilo do header */
+header {
+  width: 1200px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffffff;
+  padding: 20px;
+  margin-top: 10%;
+}
+
+/* Estilo do logo */
+.logo img {
+  max-height: 40px;
+  order:1;
+  margin-left: 80%;
+}
+
+nav {
+  order: 2; /* Define a ordem da navegação para 2 (o padrão é 0) */
+  margin-right: 5%;
+}
+
+
+/* Estilo do navegador */
+nav ul {
+  list-style-type: none;
   margin: 0;
+  padding: 0;
+}
+
+nav ul li {
```

Comment:
When avoidable don't use element's hierarchy as selectors; this creates a highly coupled code which is not reusable and hard to maintain; prefer to use classes instead